README.md

```diff
@@ -86,6 +86,13 @@ be loaded by `pdf.js`. If you want to support more browsers than Firefox you'll
 to include `compatibility.js` from `build/generic/web/`. The PDF.js files are large and
 should be minified for production.
 
+## Using PDF.js in a web application
+
+To use PDF.js in a web application you can choose to use a pre-built version of PDF.js or
+to build PDF.js from source. We supply pre-built versions for usage with NPM, Webpack and
```

Comment:
.... "version of the library or to build it from source. We supply pre-build version for usage at the npm and bower under 'pdfjs-dist' name."